pkgs/test/test/io.dart

```diff
@@ -17,7 +17,11 @@ import 'package:test/test.dart';
 /// The path to the root directory of the `test` package.
 final Future<String> packageDir =
     Isolate.resolvePackageUri(Uri(scheme: 'package', path: 'test/'))
-        .then((uri) => p.dirname(uri.path));
+        .then((uri) {
+  var dir = p.dirname(uri.path);
+  if (dir.startsWith('/C:')) dir = dir.substring(1);
```

Comment:
Can we rely on this _always_ being the `C:` drive?

Reply:
This is just for testing, so ya for now.